ocaml/typing/typedecl.ml

```diff
@@ -3219,15 +3219,17 @@ let report_error ppf = function
     fprintf ppf
       "@[[%@unboxed] attribute must be added to external declaration@ \
           argument type with layout %a. This error is produced@ \
-          due to the use of -only-erasable-extensions.@]"
+          due to the use of -extension-universe (no_extensions|\
```

Comment:
This error message won’t be helpful for our colleagues, who won’t be specifying a universe directly, but instead via a flag. It probably won’t do to just use that flag name here, but maybe saying more directly that this is because we’re trying to be upstream compatible would be helpful. (This was somewhat true before the change here, but the change here make things worse, I think.)

Reply:
Updated the errors.